test/AffInterface.purs

```diff
@@ -145,8 +145,9 @@ testGetEraSummaries = do
 
 testSubmitTxFailure :: Aff Unit
 testSubmitTxFailure = do
+  let bytes = hexToByteArrayUnsafe "00"
   runQueryM testnetTraceQueryConfig
-    (void $ submitTxOgmios (wrap $ hexToByteArrayUnsafe "00"))
+    (void $ submitTxOgmios bytes (wrap bytes))
```

Comment:
Is it just a coincidence that the hash of `"00"` is itself? Reads like a bug at first. Maybe a comment is necessary. Or does it not matter given it's a failure case?

Reply:
@jy14898 It does not matter what value we provide for the tx hash in this test, since the transaction itself is invalid